nilearn/input_data/fmriprep_confounds.py

```diff
@@ -14,38 +15,42 @@
     "motion",
     "high_pass",
     "wm_csf",
-    "global",
+    "global_signal",
     "compcor",
     "ica_aroma",
     "scrub",
-    "non_steady_state",
+    "non_steady_state"
 ]
 
 # extra parameters needed for each noise component
 component_parameters = {
     "motion": ["motion"],
     "wm_csf": ["wm_csf"],
-    "global": ["global_signal"],
+    "global_signal": ["global_signal"],
     "compcor": ["meta_json", "compcor", "n_compcor"],
     "ica_aroma": ["ica_aroma"],
     "scrub": ["scrub", "fd_thresh", "std_dvars_thresh"],
 }
 
 
-def _sanitize_strategy(strategy):
-    """Define the supported denoising strategies."""
-    if not isinstance(strategy, list):
-        raise ValueError("strategy needs to be a list of strings")
+def _check_strategy(strategy):
+    """Ensure the denoising strategies are valid."""
+    if not isinstance(strategy, tuple):
+        raise ValueError("strategy needs to be a tuple of strings"
```

Comment:
Actually it could be a list too. The mutable list is an issue for the **default value** only, otherwise lists are OK. Python has no support for the concept of `const`, we have to live with it and enforce the const'ness ourselves in our code, for example by copying a mutable before modifying it.

Reply:
Since you have already changed the default value to a tuple, I would say both tuples and lists are OK. You could double-check the list is not mutated, but since the default argument will be an immutable tuple, I am confident the list won't be mutated either if the code path is the same for both types.

Reply:
Ah thanks for explaining. That makes a lot of sense

Reply:
Looks good to me. The PEP8 error in CI seems unrelated.

Reply:
Thank you for looking into and fixing this merge request.